authority/provisioner/k8sSA_test.go

```diff
@@ -44,7 +47,24 @@ func TestK8sSA_authorizeToken(t *testing.T) {
 			return test{
 				p:     p,
 				token: "foo",
-				err:   errors.New("error parsing token"),
+				code:  http.StatusUnauthorized,
+				err:   errors.New("k8ssa.authorizeToken; error parsing k8sSA token"),
+			}
+		},
+		"fail/not-implemented": func(t *testing.T) test {
```

Comment:
I don't think it should be. It's not the same case. All those not implemented tests are because the specific provisioner does not support that method at all. In this case the provisioner does support the method. But the provisioner has been configured in a way such that it, currently, cannot support that method.